docs/KlassenTjeneste.adoc

```diff
@@ -286,7 +286,7 @@ _Norwegian extension: Not explicitly specified in CPSV-AP._
 _The value of this property is the (estimated) time needed for executing a Service._ 
 | Multiplisitet / _Multiplicity_ | 0..1 
 | Kravnivå / _Requirement level_ | Valgfri / _Optional_ 
-| Merknad 1 / _Note 1_ | Opplysningen skal oppgis ved hjelp av ISO 8601-syntaksen for varighet. Forklaring er gitt på https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia-siden &#x29C9;, window="_blank", role="ext-link"] som refererer til den offisielle ISO-standarden.
+| Merknad 1 / _Note 1_ | Opplysningen SKAL oppgis ved hjelp av ISO 8601-syntaksen for varighet. Forklaring er gitt på https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia-siden &#x29C9;, window="_blank", role="ext-link"] som refererer til den offisielle ISO-standarden.
 
 __The actual information is provided using the ISO 8601 syntax for durations. Explanation is provided in the https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia page &#x29C9;, window="_blank", role="ext-link"] that references the official ISO standard.__
```

Comment:
The information MUST be provided using....

Reply:
Kopiert og oversatt fra EU. Rettes for å være konsekvent.